src/ralph/accounts/ldap.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from dj.choices import Country
+from django.conf import settings
+from django.dispatch import receiver
+from django_auth_ldap.backend import (
+    LDAPSettings,
+    populate_user,
+)
+from django.utils.encoding import force_text
+from django_auth_ldap.config import ActiveDirectoryGroupType
+
+# Add default value to LDAPSetting dict. It will be replaced by
+# django_auth_ldap with value in settings.py and visible for each
+# ldap_user.
+LDAPSettings.defaults['GROUP_MAPPING'] = {}
+
+
+@receiver(populate_user)
+def staff_superuser_populate(sender, user, ldap_user, **kwargs):
+    user.is_superuser = 'superuser' in ldap_user.group_names
+    user.is_staff = 'staff' in ldap_user.group_names
+    user.is_active = 'staff' in ldap_user.group_names
```

Comment:
please add some comment that this is on purpose - only staff users will have access to ralph now